shared/plugin_manager.go

```diff
@@ -0,0 +1,211 @@
+package shared
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"sync"
+
+	"golang.org/x/sync/semaphore"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/go-plugin"
+)
+
+// Handshake is a common handshake that is shared by shared and host.
+var Handshake = plugin.HandshakeConfig{
+	// This isn't required when using VersionedPlugins
```

Comment:
Comment seems ambiguous. I assume we are not using VersionedPlugins, so the `ProtocolVersion` is needed?

Reply:
This is leftover comment from a copy paste action, removed.